modules/results/src/main/twirl/views/resultpanels/hhpred/forward.scala.html

```diff
@@ -1,13 +1,24 @@
 @import de.proteinevolution.models.results.FileLoader
 
-@(filepath : String)()
+@(filepath : String, jobID: String)()
 
 <div>
     <pre  id="pir" class="fileview">@FileLoader.loadFile(filepath)</pre>
     <input id="forward"  class="success button small" type="button" value="Forward to Modeller" onclick="tomodeller()">
+    <button type="button" class="button success small goToParentJob" style="display: none">Go to parent job</button>
 </div>
 <script>
     function tomodeller() {
-        Forwarding.simple("modeller", $("#pir").text())
+        Forwarding.simple("modeller", $("#pir").text(), "@jobID")
     }
+
+    $(function() {
+        var parentId = JobModel.getParamValue("parent_id");
+        if (parentId) {
+            $(".goToParentJob").show().off()
+                    .on("click", function () {
+                m.route("/jobs/" + parentId)
```

Comment:
Here, a check is missing whether the parent job still exists. If it is already deleted, this case has to be guarded by a meaningful error message